.ci/run-link-check-plugin.sh

```diff
@@ -11,9 +11,19 @@ mvn -e --no-transfer-progress clean site -Dcheckstyle.ant.skip=true -DskipTests
    -Dpmd.skip=true -Dspotbugs.skip=true -Djacoco.skip=true -Dcheckstyle.skip=true
 mkdir -p .ci-temp
 
+SKIP_EXTERNAL_LINKS=$1
+
 echo "------------ grep of linkcheck.html--BEGIN"
-grep -E "doesn't exist|externalLink" target/site/linkcheck.html | grep -v 'Read timed out' \
-  | sed 's/<\/table><\/td><\/tr>//g' | sort > .ci-temp/linkcheck-errors-sorted.txt
+LINKCHECK_ERRORS=$(grep -E "doesn't exist|externalLink" target/site/linkcheck.html \
+  | grep -v 'Read timed out' | sed 's/<\/table><\/td><\/tr>//g' || true)
+
+if [[ $SKIP_EXTERNAL_LINKS == "--skip-external" ]]; then
```

Comment:
This reads like nonsense.
Please rename SKIP_EXTERNAL_LINKS to PARAMETER

Reply:
God point, renamed to OPTION, as parameter would imply some possibly changing argument.